Makefile

```diff
@@ -8,6 +8,9 @@ all: build
 test:
 	dune runtest
 
+examples:
+	jbuilder build --dev @examples
```

Comment:
`jbuilder` → `dune`.

Reply:
Oops, missed that after the rebase. Thanks